Src/EB/AMReX_EB2_Level.H

```diff
@@ -496,6 +499,11 @@ GShopLevel<G>::define_fine (G const& gshop, const Geometry& geom,
                 }
             }
         }
+        total_multicuts = total_multicuts + nmulticuts;
+    }
+    if (plt_multiple_cuts){
+        amrex::Print() << "Grand total multicuts = " << total_multicuts << "\n";
+        amrex::Abort("EY: amrex::EB2::build_faces: more than 2 cuts not supported"); // move this 
```

Comment:
Remove `EY: ` from the abort message. In general it's not needed in this situation or in comments because if people want to find who wrote some piece of code they can use git blame.

Reply:
Yeah I was going to remove them all before we actually make a pull request 😂